simplejira/tables.py

```diff
@@ -38,8 +38,9 @@ def test_row(self, attribute, value):
         valid = False
         if isfunction(value):
             try:
-                value(self.entries[0])
-                valid = True
+                if self.entries:
```

Comment:
This was necessary otherwise it would fail with index out of bound error for empty lists.